pkg/generated/clientset/versioned/fake/register.go

```diff
@@ -45,14 +45,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//   import (
-//     "k8s.io/client-go/kubernetes"
-//     clientsetscheme "k8s.io/client-go/kubernetes/scheme"
-//     aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
-//   )
+//	import (
+//	  "k8s.io/client-go/kubernetes"
+//	  clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+//	  aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
+//	)
```

Comment:
@owenthereal what is your golang version installed. For 1.17 and 1.18 generated files should not be changed

Reply:
I'm using 1.19 locally. I regenerated the code with 1.18 in https://github.com/zalando/postgres-operator/pull/2161/commits/3fb0b42f27c025ffe9e8088088ab0007262264f8. 

It would be nice to introduce https://asdf-vm.com or the like to this project to auto-switch or prompt to install the right Go version.

Reply:
Not sure we need an additional tools for that, as golang supports multiple versions management by itself
https://go.dev/doc/manage-install

As an option, some validation steps with autogenrration/formatting/linting can be added as part of the #2135